db/migrate/20171009151031_add_given_order_to_poll_question_answers.rb

```diff
@@ -0,0 +1,5 @@
+class AddGivenOrderToPollQuestionAnswers < ActiveRecord::Migration
+  def change
+    add_column :poll_question_answers, :given_order, :integer
```

Comment:
Nice one! I started a quick refactor to add this but found myself in the pitfall of creating an column named `order` that raised many issues 🙈  this name is better

Although by reading all code seems to me that the `given_order` values will always start from `1`, right? Would it make sense to set a `default: 1` just as a reminder on the code and as well as a security measure?

Also I think if `given_order` is an attribute needed there should be a presence validation on the model (as well as one that checks its unique for the question_id (only one Answer on a particular order for its Question)